### ISSUE TITLE
Aggregate Artsdata source (capitol-nb-ca) for ArtscultureNB

### ISSUE THREAD

**Issue by tammy-culture** [culturecreates/footlight-app#1288]
Please aggregate the following Artsdata sources : [http://kg.artsdata.ca/culture-creates/artsdata-orion/capitol-nb-ca]

**Comment by sahalali** [comment]
I made a workflow to import events and linked entities from capitol-nb-ca. However, none of the events have a French or English label, causing a problem with the listing.

@saumier Do you think it would be helpful to add support for automatically copying the label into the aggregator if no English/French label is available? We can do this by adding an extra configuration for a fall-back label.

**Comment by tammy-culture** [comment]
@saumier What are next steps here so I understand.  I see that there are no titles in the Production CMS.  Do I need to get ArtscultureNB (Benjamin) to speak to Capitol Theatre to change something in JSON-LD on their website?

**Comment by saumier** [comment]
@tammy-culture No. We need to wait for CMS to support the names without any specific language tag.

**Comment by sahalali** [comment]
Workflow is ready and scheduled for every thursdays
The events under this graph do not have an associated image in the Artsdata.

Example: https://kg.artsdata.ca/entity.html?uri=http%3A%2F%2Fkg.artsdata.ca%2Fresource%2FK43-749

**Comment by tammy-culture** [comment]
@troughc I see that I am unassigned, is there anything that I need to do? Should I let the client (artscutureNB) know that an image is not available in Artsdata?

**Comment by troughc** [comment]
@tammy-culture it is in @saumier hands right now. Please check with him.

**Comment by sahalali** [comment]
@saumier  Reminder

**Comment by tammy-culture** [comment]
@saumier  As per our discussion Capitol-NB JSON-LD has no time zones and therefore times are loading incorrectly.  @saumier To investigate whether this can be fixed in Artsdata.ca

If still unresolved by March 1st, @tammy-culture will use Footlight Console.
